Source/Core/VideoBackends/OGL/OGLRender.cpp

```diff
@@ -513,6 +513,9 @@ Renderer::Renderer(std::unique_ptr<GLContext> main_gl_context, float backbuffer_
     // ARB_get_texture_sub_image (unlikely, except maybe on NVIDIA), we can use that instead.
     g_Config.backend_info.bSupportsDepthReadback = g_ogl_config.bSupportsTextureSubImage;
 
+    // GL_TEXTURE_LOD_BIAS is not supported on GLES.
+    g_Config.backend_info.bSupportsLodBiasInSampler = false;
+
```

Comment:
It looks like you're only ever setting it to false for GLES; you're never setting it to true for regular OpenGL.  That's probably the cause of the fifoci differences on ogl-lin-mesa.

Reply:
I believe I set it to true by default in ``OGLMain.cpp``?

Reply:
Ah, yes, I missed that.  And FifoCI does seem to be using regular OpenGL (see [the build log](https://dolphin.ci/#/builders/16/builds/5337/steps/7/logs/stdio), which says `Dolphin [refs/pull/10292/head] 5.0-15686 | FifoPlayer SC | OpenGL | HLE | FPS: 1 - VPS: 1 - 0%` instead of `OpenGL ES`).